starter/src/app/_components/home/filters.tsx

```diff
@@ -0,0 +1,73 @@
+"use client";
+
+/* */
+import Results from "@/app/_components/home/results";
+import SidebarItem from "@/app/_components/home/sidebar-item";
+import SignupCard from "@/app/_components/home/signup-card";
+import { useBreakpoint } from "@/app/_components/universal/hooks/use-breakpoint";
+import { Layout } from "@/app/_components/universal/layout";
+import { sidebarCategories, resultItems } from "@/app/constants";
+import type { SidebarCategory, ResultItem } from "@/app/constants";
+import { Button } from "@/components/ui/button";
+import {
+  Dialog,
+  DialogPortal,
+  DialogOverlay,
+  DialogClose,
+  DialogTrigger,
+  DialogContent,
+} from "@/components/ui/dialog";
+
+export const HomeFilters = () => {
+  const isMd = useBreakpoint("md");
+  if (isMd)
+    return (
+      <Layout flex fullWidth={false}>
+        <Layout.Aside>
+          {sidebarCategories.map(({ title, items }: SidebarCategory) => (
+            <SidebarItem key={title} title={title} items={items} />
+          ))}
+        </Layout.Aside>
+        <Layout.Main>
+          <div className="grid grid-cols-2 gap-4 space-x-2 xl:grid-cols-3">
+            <SignupCard />
+            <Results items={resultItems} />
+          </div>
+        </Layout.Main>
+      </Layout>
+    );
+  return (
+    <>
+      <Layout flex fullWidth={false} align="end">
+        <div className="dialog flex flex-row-reverse px-4">
+          <Dialog modal={false}>
+            <DialogTrigger asChild>
+              <Button className="openFilters">Open Filters</Button>
```

Comment:
```suggestion
            <Button
                variant="outline"
                size="sm"
                className="h-7 gap-1 text-sm"
            >
                <ListFilter className="h-3.5 w-3.5" />
                <span className="sr-only sm:not-sr-only">Filter</span>
            </Button>
```

Reply:
don't accept as this isn't properly prettier'd

Just a proposal

Reply:
yes, you are right, that I also left for the polish part

just wanted to merge files changes sooner for not having conflicts